### PR TITLE
QRadar documantation and change default to 30

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Added troubleshoot section
- Change default of polling command to 30 seconds

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 1]
```diff
@@ -100,6 +100,15 @@ Note: If you're uncertain which API version to use, it is recommended to use the
...
 2. Click the **Menu** toolbar. A scrolling toolbar will appear.
 3. Click **About**. A new window will appear with the details of your QRadar version.
+
+## Troubleshooting
+
+Whenever *Fetch with events* is configured, the integration will fetch the offense events from `QRadar`.
+It is possible, however, that some events might be missed during the creation of the incident.
+The [Mirroring](#mirroring-events) feature is recommended in this case.
+Alternatively, [Retrieve events command](#qradar-search-retrieve-events) can be used to retrieve the `QRadar` immediately.
+If the command does not execute, try setting the `interval_in_seconds` to 10.
```
Comment (tkatzir) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:110]:
```suggestion
If the command does not execute, try setting the `interval_in_seconds` to a lower value, like 10.
```

**Review by tkatzir — APPROVED** [review]
Pending my wording change request.

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 2]
```diff
@@ -100,6 +100,15 @@ Note: If you're uncertain which API version to use, it is recommended to use the
 1. Enter QRadar service.
 2. Click the **Menu** toolbar. A scrolling toolbar will appear.
 3. Click **About**. A new window will appear with the details of your QRadar version.
+
+## Troubleshooting
+
+Whenever *Fetch with events* is configured, the integration will fetch the offense events from `QRadar`.
+It is possible, however, that some events might be missed during the creation of the incident.
+The [Mirroring](#mirroring-events) feature is recommended in this case.
```
Comment (DeanArbel) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:108]:
```suggestion
It is recommended to enable [mirroring](#mirroring-events), as it should fetch previously missed events when the offense is updated.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 3]
```diff
@@ -100,6 +100,15 @@ Note: If you're uncertain which API version to use, it is recommended to use the
...
+Alternatively, [Retrieve events command](#qradar-search-retrieve-events) can be used to retrieve the `QRadar` immediately.
```
Comment (DeanArbel) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:109]:
```suggestion
Alternatively, the [retrieve events command](#qradar-search-retrieve-events) can be used to retrieve the `events` immediately.
```

File: Packs/QRadar/Integrations/QRadar_v3/README.md [hunk 4]
```diff
@@ -100,6 +100,15 @@ Note: If you're uncertain which API version to use, it is recommended to use the
...
+If the command does not execute, try setting the `interval_in_seconds` to a lower value, like 10.
```
Comment (DeanArbel) [inline Packs/QRadar/Integrations/QRadar_v3/README.md:110]:
```suggestion
If the command takes too long to finish execution, try setting the `interval_in_seconds` to a lower value (down to a minimum of 10 seconds).
```

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py [hunk 5]
```diff
@@ -3449,7 +3449,7 @@ def qradar_search_retrieve_events_command(client: Client,
     Returns:
         CommandResults: The results of the command.
     """
-    interval_in_secs = int(args.get('interval_in_seconds', 60))
+    interval_in_secs = int(args.get('interval_in_seconds', 30))
```
Comment (DeanArbel) [inline Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py:3452]: Let's represent this value via the `yml` default value, so it'll be transparent to the user.

**Review by DeanArbel — APPROVED** [review]
Nice. Please see my comments below.

**Comment by julieschwartz18** [comment]
Doc review complete for the 2_2_6.md file. @ilaner - please confirm:
Is the command name qradar-search-retrieve-command or qradar_search_retrieve_events_command ?

**Pull request merged**